Code/client/Services/Generic/ObjectService.cpp

```diff
@@ -96,7 +96,7 @@ void ObjectService::OnCellChange(const CellChangeEvent& acEvent) noexcept
 
     // Player homes should not be synced, so that chest contents,
     // which are often used as storage, are never accidentally wiped.
-    if (IsPlayerHome(pCell))
+    if (IsPlayerHome(pCell) && !World::Get().GetServerSettings().SyncPlayerHomes)
```

Comment:
It's more efficient to check for the SyncPlayerHomes variable before calling IsPlayerHome